pkg/api/tmpl/chi-middleware.tmpl

```diff
@@ -0,0 +1,190 @@
+// ServerInterfaceWrapper converts contexts to parameters.
```

Comment:
Add a link to the original template and also an explanation of what changed, so it's easier to keep track of changes.